README.md

```diff
@@ -19,175 +21,8 @@ The NTT manager contracts additionally keep track of the token decimals of the o
-
-The following will be produced in the program logs:
-
-```
-Program log: Instruction: ReceiveMessage
-```
-
-[`redeem`] checks the inbound rate limit and places the message in an Inbox. Logic works the same as the outbound rate limit we mentioned previously.
-
-The following will be produced in the program logs:
-
-```
-Program log: Instruction: Redeem
-```
-
-5. Mint or Unlock
-
-The inbound transfer is released and the tokens are unlocked or minted to the recipient (depending on the mode) through either [`release_inbound_mint`] (if the mode is `burning`) or [`release_inbound_unlock`] (if the mode is `locking`). Similar to transfer, using the wrong transfer instruction, i.e. [`release_inbound_mint`] for a program that is in "locking" mode, will result in `InvalidMode` error.
-
-> When `revert_on_delay` is true, the transaction will revert if the release timestamp has not been reached. When `revert_on_delay` is false, the transaction succeeds, but the minting/unlocking is not performed.
-
-Depending on the mode and instruction, the following will be produced in the program logs:
-
-```
-Program log: Instruction: ReleaseInboundMint
-Program log: Instruction: ReleaseInboundUnlock
-```
-
-#### Installation
-
-Install [Foundry](https://book.getfoundry.sh/getting-started/installation)
-[Foundry]: https://book.getfoundry.sh/getting-started/installation#using-foundryup
-
-TODO: add installation instructions for solana
-
-Install [rust](https://doc.rust-lang.org/book/ch01-01-installation.html)
-
-```
-curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
-```
-
-#### Developer Commands
-
-_Build_
-
-```
-$ forge build
-```
-
-_Test_
-
-```
-$ forge test
-```
-
-#### Submitting a PR
-
-Before submitting a PR, please run the following commands:
+## Rate-Limiting and Cancel Flows
 
-_Test_
-EVM Tests:
+NTT supports rate limiting on the sending. If a transfer is rate limited on the source chain and queueing is enabled, transfers are placed into a outbound queue that can be released after the expiry of the rate limit duration.Transfers that are rate-limited on the destination chain are added to an inbound queue.
 
-```
-$ make test-evm
-```
+If users bridge frequently between a given source chain and destination chain, the capacity could be exhausted quickly. This can  leave other users rate-limited, potentially delaying their transfers. To mitigate this issue, inbound rate limit is refilled on the source chain by an amount equal to that of the outbound transfer amount and vice-versa, the outbound rate limit is refilled on the destination chain by an amount equal to the inbound transfer amount. We refer to this colloquially as "Cancel Flow".
```

Comment:
I would make the cancel flow a heading here, and actually use the word cancel in the explanation (something like an inbound transfer can cancel out the outbound rate limit and vice versa)